### PR TITLE
folder changes and route constants changed

### PR DESCRIPTION
**Pull request by joesakkab**

### CODE REVIEW

File: client/src/components/Search/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  },
+  button: {
+    width: '100%',
+    margin: theme.spacing(1),
+    backgroundColor: '#2196f3', // Set the button background color to match the background color of the page
+    color: '#fff', // Set the button text color to white
+  },
+}));
+
+function Landing() {
```
Comment (ammar-s847) [inline client/src/components/Search/index.js:47]: This could be a different PR, but otherwise looks good

**Review by ammar-s847 — APPROVED** [review]

File: client/src/components/Search/index.js (hunk 1 above)
Comment (joesakkab) [reply client/src/components/Search/index.js:47]: thankyouuuu
